kushaldas/johnnycanencrypt
# Db upgrade

## Idea

We will create a fresh temporary db, and then reimport all the existing keys into. Then we sync the values of `oncard` for each key.

## How to test?

First create a `jce.db` file from the `master` branch via a keystore object and do the same from this branch. It should not raise any exception. The database should have the revoked status of each key.

Check the newly added test cases for more details.
